### PR TITLE
Test for `round_to_2()`

### PR DESCRIPTION
**Pull request by brycemcd**
I've been reading more and more about exoplanets and have looking for a way to get involved. I like this project and hope I can help out a little.

This PR adds some documentation and tests to a function in the `utils` package. If this is desired I can continue to add more tests and more documentation.

I've been a software engineer for a long time but have been a hobbyist astronomer for only a year or so. My day job will prevent me from being a prolific contributor to this project but I am happy to use pockets of time I have to writing tests and making the code more pythonic and idiomatic. I want to support the work you're all doing :)

### CODE REVIEW

**Comment by brycemcd** [comment]
Chronicling a conversation from #869 to make sure the maintainers understand the approach I have been considering for this PR and the others that mention this one:

Once this PR is merged then some code that sets up testing for this codebase will be merged. I can rebase the WIP MRs on the new `develop` branch and remove the WIP from the PR. That should be quick. Those PRs that are open will then just have code limited to the functions they test and will be easier to review.

File: exotic/tests/test_utils.py
Comment (rzellem) [inline exotic/tests/test_utils.py:51]: doesn't look like a bug to me- it takes the second number, looks for 2 sig figs (non-0s after the decimal), so in this case that would be 0.00012, and then rounds the first number to that number of sig figs - which should be 2.12300 - but the trailing 0s get dropped by Python

File: exotic/tests/test_utils.py
Comment (rzellem) [inline exotic/tests/test_utils.py:1]: why the new tests folder?

**Review by rzellem — CHANGES_REQUESTED** [review]

File: exotic/tests/test_utils.py
Comment (brycemcd) [reply exotic/tests/test_utils.py:1]: It's a way to organize tests that keeps the tests 1) close to the code and 2) the implementation code uncluttered.

Documentation for python-conventional ways to organize tests is here: https://docs.pytest.org/en/6.2.x/goodpractices.html#choosing-a-test-layout-import-rules I chose the approach that made the most sense.

File: exotic/tests/test_utils.py
Comment (brycemcd) [reply exotic/tests/test_utils.py:51]: good to know. I wasn't sure if more precision would be needed. I'll update to remove this comment.

File: exotic/tests/test_utils.py
Comment (brycemcd) [reply exotic/tests/test_utils.py:1]: There are quite a few open source, popular, python repositories that use this paradigm of testing. Check out [qlib](https://github.com/microsoft/qlib) and it's [top level test directory](https://github.com/microsoft/qlib/tree/main/tests), [sklearn](https://github.com/scikit-learn/scikit-learn/tree/main/sklearn) and [numpy](https://github.com/numpy/numpy/tree/main/numpy) to see how they've implemented their test suites. I used these to guide this and my other testing branches.

I'm happy to chat through other paradigms if you would like. I'm also happy to change paradigms in the future (and backport these) as we get more testing code implemented and a different paradigm reveals itself as advantageous.

**Review by rzellem — APPROVED** [review]

**Pull request merged**
